onnx/version_converter/adapters/backwards_compatible.h

```diff
@@ -0,0 +1,19 @@
+// Adapter indicating backwards compatibility of op between opsets with
+// separate definitions
+
+#pragma once
+
+#include "onnx/version_converter/adapters/adapter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+struct BackwardsCompatibleAdapter final : public Adapter {
+  // TODO: Should these be &&?
+  explicit BackwardsCompatibleAdapter(const std::string& op_name, const OpSetID
```

Comment:
Again, the std::move should not combine with const

Reply:
More detail on protocol: const references for strings, but move semantics for other objects?